wd_capabilities.au3

```diff
@@ -331,27 +361,6 @@ Func __WD_CapabilitiesInitialize($s_MatchType, $s_BrowserName = '') ; $s_MatchTy
 	#EndRegion - FOR TESTING ONLY
 EndFunc   ;==>__WD_CapabilitiesInitialize
 
-; #INTERNAL_USE_ONLY# ===========================================================================================================
-; Name ..........: __WD_CapabilitiesSwitch
```

Comment:
as   __WD_CapabilitiesSwitch()  is removed 
should be check how variables are switched

Reply:
Are fine

Reply:
> as __WD_CapabilitiesSwitch() is removed


Is this should be documented in change log ?

Reply:
I don't think it is necessary because it was for internal use only.

Reply:
ok